### DISCUSSION TITLE
Show output in JSON format

### DISCUSSION THREAD

**Discussion by pgaijin66** [go-task/task#1177] [Ideas]
> Describe in detail what feature do you want to see in Task.

Thank you so much for this project. I have been using Taskfile in my personal projects and it has made my life much easier and effecient. What i was thinking was It would be really awesome to be able to parse task output in JSON and use it in conjunction with other tools like `jq`. Since we are listings task info as json anyway. 

> Give examples if possible.

For a given task eg:
```
version: '3'

tasks:
  run:
    cmds:
      - echo "hello world"
```
If we could do something like this
`task run --json`

output:
```
{
  "tasks": [
    {
      "name": "Build go app",
      "stdout": ["hello world"],
      "stderr": [],
    }
  ],
  "location": "/home/demo/Taskfile.yml"
}
```
>

**Comment by andreynering** [comment]
Hi @pgaijin66,

No plans to implement something like that at the moment.
